modules/programs/rofi-pass.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.rofi.pass;
+in {
+  meta.maintainers = [ maintainers.seylerius ];
+
+  options.programs.rofi.pass = {
+    enable = mkEnableOption "Enable rofi integration with password-store";
```

Comment:
dont need the `Enable` since mkEnableOption prepends it

Reply:
Thanks for the tip, fixed in 27f66d3.